lib/declarations.d.ts

```diff
@@ -543,16 +586,75 @@ interface IPluginsService {
 	 * @return {IBasicPluginInformation}            Basic information about the plugin
 	 */
 	getPluginBasicInformation(pluginName: string): IBasicPluginInformation;
+
+	/**
+	 * Copies the source code of a plugin inside the project and adds it as a reference, so it can be used within the application.
+	 * @param {string[]} pluginIdentifiers The identifiers of the plugin that will be copied to the source code.
```

Comment:
small typo - `The identifiers of the plugin that will be copied to the source code.` should be something like `The identifiers of the plugins that will be copied to the source code.`